Examples/Io/Csv/src/CsvOutputData.hpp

```diff
@@ -135,25 +135,32 @@ struct SurfaceData {
   /// Surface identifier. Not available in the TrackML datasets.
   uint64_t geometry_id;
   /// Partially decoded surface identifier components.
-  uint32_t volume_id, layer_id, module_id;
+  uint32_t volume_id, boundary_id, layer_id, module_id;
   /// Center position components in mm.
   float cx, cy, cz;
   /// Rotation matrix components.
   float rot_xu, rot_xv, rot_xw;
   float rot_yu, rot_yv, rot_yw;
   float rot_zu, rot_zv, rot_zw;
-  /// Limits and pitches in mm. Not always available.
-  float module_t = -1;
-  float module_minhu = -1;
-  float module_maxhu = -1;
-  float module_hv = -1;
-  float pitch_u = -1;
-  float pitch_v = -1;
```

Comment:
Pitch is not stored anymore correct? This makes the geometry file only contain the actual geometry and not digitization information, right? I am Ok with that but I want to clarify that that is your intention.

Reply:
It is my intention, the segmentation should be dealt differently (in a different file).